logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -76,51 +75,25 @@ data class EventEnvelope(
 /**
  * Data class representing information about the delivery of an event.
  *
- * @property isTransient Specifies whether the event is transient.
- * Transient events are events that only matter if the user is online/active. For example "user is typing",
- * and call signaling (mute/unmute), which are irrelevant after a few minutes. These are likely to not even
- * be stored in the backend.
- * @property source The source of the event.
- * @see EventSource
+ * @property isLive Indicates whether the event was received in real-time via WebSocket (`true`)
+ * or fetched in batch as a pending event (`false`).
  */
-sealed class EventDeliveryInfo(
-    open val isTransient: Boolean,
-    open val source: EventSource
+data class EventDeliveryInfo(
+    val isLive: Boolean,
```

Comment:
Might as well use the `EventSource` to improve readability.
I see some places doing `EventDeliveryInfo(true)`, but `EventDeliveryInfo(EventSource.LIVE)` is more readable.